xgi/core/simplicialcomplex.py

```diff
@@ -745,12 +745,12 @@ def remove_simplex_id(self, id):
         remove_edges_from : remove a collection of edges
         """
         try:
-            # remove all simplices that contain simplex
+            # Remove all simplices that contain the given simplex
             supfaces_ids = self._supfaces_id(self._edge[id])
             for sup_id in supfaces_ids:
-                self._remove_simplex_id(sup_id)
+                self.remove_simplex_id(sup_id)
```

Comment:
I don't this was wrong before so I wouldn't touch it, unless you think it's more efficient?

Reply:
good catch, thx